core/common/src/main/java/alluxio/util/io/BufferUtils.java

```diff
@@ -51,6 +55,53 @@ public static int byteToInt(byte b) {
    * this direct buffer should be discarded. This is unsafe operation and currently a work-around to
    * avoid huge memory occupation caused by memory map.
    *
+   * @param buffer bytebuffer
+   */
+  public static synchronized void cleanDirectBuffer(ByteBuffer buffer) {
```

Comment:
What was the reason for making everything `synchronized`? Is it for initializing the method variable, or is it also for the actual cleaning part?

Reply:
it was synchronized before. I think the reason is that we initialize static variable within the method, so it has to be synchronized. cleaning part does not require synchronization.